jina/drivers/rank/__init__.py

```diff
@@ -66,7 +65,7 @@ def _sort_matches_in_place(self, context_doc: 'Document', match_scores: 'np.ndar
         cm = context_doc.matches
         cm.build()
         for str_match_id, score in match_scores:
-            match_id = UniqueId(str_match_id)
+            match_id = str_match_id
```

Comment:
iterate directly on match id

Reply:
you are so right! Thanks. This statement is not needed anymore